requirements.txt

```diff
@@ -1,4 +1,9 @@
 pycryptodome
-lxml
-Pillow
-aiohttp
\ No newline at end of file
+lxml~=4.5.1
+Pillow~=7.1.2
+aiohttp~=3.6.2
+pyexiv2~=2.2.0
+cssselect
+unidecode
+pandas
+html
```

Comment:
It would be good if we could avoid multipying the dependencies

Reply:
Sorry - mind to clarify? I'm not familiar with dependencies so not sure what I did wrong here.

Reply:
Can we remove unidecode and pandas and not make cssselect and html mandatory ? We can try to import them and disable the corresponding features when they are not available.

Reply:
Should be done in latest commit... but please let me know if I missed something.

Reply:
Any idea of how I would make cssselect conditional? I thought it was needed for scraping the metadata. As far as I know, every Arts&Culture page has metadata that can be scraped (if not for embedding, at least for final name formatting)